community/code-of-conduct.md

```diff
@@ -0,0 +1,166 @@
++++
+
+toc=true
+
++++
+
+# [Name Option 1] Workplace Harassment - Code of Conduct 
+# [Name Option 2] An informal guide to avoid and deal with events of harassment and bullying
+# [Name Option 3] Workplace Harassment - An informal guide
+
+## Some basic fundamentals
+
+1. Humans are flawed and make mistakes on a regular basis while discovering new ways to hurt new people.
+
+2. We don't live in an ideal world, so we are not dealing with ideal scenarios.
+
+3. But the above statement doesn't provide wind to the practicality argument. Practicality argument is an exceptionally dangerous proposition in itself. So any time any particular statement is backed up by "being practical", it should take care of few things :-
+   1. Being practical means what can be done or is possible in some finite duration. So it should be aggressively progressive in nature and it shouldn't settle on "n" but "n + ∞". We can start from "n" but the vision should be clear to reach "n + ∞" as fast as possible.
+   2. It should be “enabler” in nature.
+
+4. Any democratic argument in case of harassment should be thrown out of the window immediately. For example, "9 out of 10 people are okay with this".
+
+5. Humans are not rational, so don't expect them to behave rationally. We are totally full of feelings.
+
+6. We don't accept snowflake argument. Some people have thick skin and are borderline stoic, and they think that "I am responsible for my feelings" and on the opposite end of emotional spectrum, are people who think that "You don't get to decide if you hurt someone". We will try to have an inclusive space for both of these people.
+
+7. Humans hate to forgive people if forgiving is tough.
+
+8. Convincing a human that they are wrong or have done wrong is exceptionally tough.
+
+9. Humans have double standards. Deal with it.
+
+## Basic Protocol in case you are harassed/bullied/found something offensive/weird/unacceptable
+
+Subject -> Who has been harassed/bullied
+
+Type 1 :- Constitutionally Legal Offense
```

Comment:
Since we will be, except in some extreme unlikely case, dealing with KGP students mentioning ICC will help.

Ref- http://www.iitkgp.ac.in/internal-complaints-committee

Reply:
I am a little skeptical of adding the Internal Complaints Committee. I have heard of an incident where they have intentionally buried  a case to preserve the "image" of the Insti.

Also ICC is specifically for the "employees", I don't know if they cover students legally.

> This cell undertakes inquiries with regard to complaints of sexual harassment from aggrieved women employees (as defined under the Act, 2013). 

Although it is up to the victim to choose which path to follow, we should be careful on what to include.

Reply:
Accha. Makes sense to not want to include as they'll try to suppress the matter. However, AFAIK, in practicality no matter where you make the complaint regarding the incident (in KGP), it'll be routed to the institute and they'll want to save face :cry:

Reply:
No, ICC is also for female students. I know one friend who has dealt with a situation with the help of ICC. It is there "to effectively deal with cases of violence against female students, staff, faculty and campus residents.". 
Though I wouldn't be surprised to know that an incident is buried by the administration in favour of "saving the image of the institution". Regardless of that, I do believe that we should be mentioning about ICC in this document.